moveit_core/robot_model/include/moveit/robot_model/joint_model_group.h

```diff
@@ -282,7 +282,7 @@ class JointModelGroup
   }
 
   /** \brief Get the index locations in the complete robot state for all the variables in this group */
-  const std::vector<int>& getVariableIndexList() const
+  const std::vector<size_t>& getVariableIndexList() const
```

Comment:
I hope there is no code that does some "clever" negative value index tricks (like use -1 as a flag of a kind or something similar).

Reply:
There was and I removed it with a previous PR.